src/ConvertMD.ts

```diff
@@ -0,0 +1,206 @@
+                [key: string]: {
+                    $ref: string;
+                };
+            };
+            messages?: {
+                [key: string]: {
+                    $ref: string;
+                };
+            };
+        };
+    };
+    components?: {
+        messages?: {
+            [key: string]: {
+                name: string;
+                title: string;
+                summary: string;
+                contentType: string;
+                traits?: {
+                    [key: string]: {
+                        $ref: string;
+                    };
+                };
+                payload?: {
+                    $ref: string;
+                };
+            };
+        };
+        schemas?: {
+            [key: string]: {
+                type: string;
+                properties: {
+                    [key: string]: {
+                        type: string;
+                        minimum?: number;
+                        maximum?: number;
+                        description?: string;
+                        enum?: string[];
+                        format?: string;
+                        items?: {
+                            type: string;
+                        };
+                        $ref?: string;
+                    };
+                };
+                description?: string;
+            };
+        };
+        securitySchemes?: {
+            [key: string]: {
+                type: string;
+                description: string;
+            };
+        };
+        parameters?: {
+            [key: string]: {
+                description: string;
+                schema: {
+                    type: string;
+                    format?: string;
+                    minimum?: number;
+                    maximum?: number;
+                };
+            };
+        };
+        messageTraits?: {
+            [key: string]: {
+                headers: {
+                    type: string;
+                    properties: {
+                        [key: string]: {
+                            type: string;
+                            minimum?: number;
+                            maximum?: number;
+                        };
+                    };
+                };
+            };
+        };
+        operationTraits?: {
+            [key: string]: {
+                bindings: {
+                    kafka: {
+                        clientId: {
+                            type: string;
+                            enum: string[];
+                        };
+                    };
+                };
+            };
+        };
+    };
+}
+export function convertAsyncAPIToMermaid(asyncAPIFilePath:string): string {
+  const yamlFile = fs.readFileSync(asyncAPIFilePath, 'utf8');
+  const asyncAPIDocument: AsyncAPIDocument = yaml.load(yamlFile) as AsyncAPIDocument;
+  let mermaidCode = `flowchart TD\n`;
+
+  mermaidCode += `    subgraph "${asyncAPIDocument.info.title}"\n`;
```

Comment:
this is a good start but I should evolve to use some templating engine for js
I have used https://ejs.co/ for this kind of templates, but feel free to propose any other one..